packages/api/src/wasi.rs

```diff
@@ -465,14 +465,14 @@ impl Environment {
     /// import_object = wasi_env.generate_import_object(store, wasi.Version.SNAPSHOT1)
     /// ```
     //#[pyo3(text_signature = "($self, store, wasi_version)")]
-    fn generate_import_object(&self, store: &Store, wasi_version: Version) -> ImportObject {
+    fn generate_import_object(&self, store: &Store, wasi_version: Version) -> PyResult<PyObject> {
```

Comment:
Isn't this a breaking change? If we're OK making breaking changes then we should eliminate `ImportObject` from the python API entirely and just use dict everyone.

Reply:
Yeah, this is a slight small breaking change (on the return type), but all APIs behave the same.

I was thinking on removing the ImportObject, but if we do that we will make invalid a lot of existing code. I think we can just mark it as deprecated for now.

Reply:
I would prefer if we had a separate function which returns a dict, while keeping the existing one. This would avoid any breaking changes.

We can always clean this up later when we make a new release with breaking changes (3.0?).

Reply:
Sounds reasonable. Can you handle this from here?

Reply:
Sure.